Automattic/newspack-blocks
# Donate Block: Update the colour variables and remove Muriel

### All Submissions:






### Changes proposed in this Pull Request:

* Remove the use of legacy (Muriel) colours
* Use the same variables as the Newspack Theme
* Adjust the border for `.wpbnbd` and `.donation-frequency-label`

### How to test the changes in this Pull Request:

1. Add the donation block to a page
2. Visit the page
3. Switch to this branch
4. Refresh the page, do you still see the same(-ish) colours?

### Other information:
